packages/app/src/systems/Bridge/services/bridge.tsx

```diff
@@ -69,9 +81,17 @@ export class BridgeService {
       });
 
       if (txId) {
-        store.openTxEthToFuel({
-          txId,
-        });
+        if (fuelWallet) {
```

Comment:
```suggestion
        if (txId && fuelWallet) {
```
You don't need to have to nested ifs in this case

Reply:
it's not the same logic, look the `return;  ` in the lines below

Reply:
In this case the return is outside of the `f (txId)`